exercises/simple-cipher/.meta/hints.md

```diff
@@ -0,0 +1,35 @@
+## Implementation
```

Comment:
I really like the "## Implementation" section for hints.md.  It works very well to bridge from the description content.

Reply:
Me too. if we find it useful we may want to put it in the template.